sonar-softvis3d-plugin/src/main/resources/static/overview_page.js

```diff
@@ -0,0 +1,58 @@
+window.registerExtension('softvis3d/overview_page', function (options) {
```

Comment:
This looks very interesting:
Instead of exposing the options onto the window element, we could instead instantiate our app directly here and get rid of the `index.ts`

Something like this
```ts
window.registerExtension('softvis3d/overview_page', function (options) {
    let div = document.createElement('div');
    div.setAttribute('id', 'app');
    options.el.appendChild(div);

    initApp = function() {
        // Extract options directly from SonarQube and inject them into the app
        const softvis3d = new App({
            api: options.get.api.url,
            projectKey: options.component.key,
        });
        softvis3d.run("app");
    }

    loadStyle(window.baseUrl + '/static/softvis3d/style.css');
    loadScript(window.baseUrl + '/static/softvis3d/react.min.js', function () {});
    loadScript(window.baseUrl + '/static/softvis3d/react-dom.min.js', function () {});
    loadScript(window.baseUrl + '/static/softvis3d/three.min.js', function () {
        loadScript(window.baseUrl + '/static/softvis3d/bundle.js', initApp);
    });
});
```